protobuf/types/struct.proto

```diff
@@ -0,0 +1,115 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
A link to original file should be included: https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/struct.proto
```suggestion
// Source: https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/struct.proto
// Protocol Buffers - Google's data interchange format
```